internal/provider/resource_tfe_workspace_test.go

```diff
@@ -2293,6 +2296,20 @@ func testAccCheckTFEWorkspaceHasRemoteConsumers(ws string, wsConsumers []string)
 	}
 }
 
+// Helper to to stub attribute checks for auto_apply_run_trigger until it's supported in TFE.
+func testAccCheckTFEWorkspaceAutoApplyRunTrigger(res, value string) resource.TestCheckFunc {
+	if enterpriseEnabled() {
+		return func(s *terraform.State) error {
+			return nil
+		}
+	}
+	return resource.TestCheckResourceAttr(res, "auto_apply_run_trigger", value)
+}
```

Comment:
This is a fantastic helper. I'm wondering if we could refactor this method as a generic helper for any attribute we should test TFC-only. This could avoid having to duplicate this logic again or worse, force the parent test to be flagged entirely. 👍

Reply:
👀 that's an interesting thought. I'll be going back to this branch anyway to update go-tfe, may as well refactor this to a general helper while I'm in there.